packages/client/components/NewMeetingRoot.tsx

```diff
@@ -19,9 +21,30 @@ const NewMeetingRoot = () => {
   useSubscription('NewMeetingRoot', TaskSubscription)
   useSubscription('NewMeetingRoot', TeamSubscription)
   const queryRef = useQueryLoaderNow<NewMeetingQuery>(newMeetingQuery, {teamId})
-  return (
+
+  const location = useLocation<{backgroundLocation?: Location}>()
+  const history = useHistory()
+
+  const onClose = useCallback(() => {
+    const state = location.state
+    history.push(state?.backgroundLocation ?? '/meetings')
```

Comment:
-1 as convention, it's nice to use `history.replace` for modals. typically when you hit the browser back button you expect more than to close the modal